docs-src/content/introduction/getting-started.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Getting started"
+weight: 020
```

Comment:
Would you consider getting started the first item before stepping into features?

Reply:
Yep, better, I'll merge and we can change this in the following PRs